### PR TITLE
e2e org page test

### PR DESCRIPTION
**Pull request by HammerMeetNail**
* Adds e2e tests for org page

### CODE REVIEW

**Comment by HammerMeetNail** [comment]
@bcaton85 Looks like the repository list tests are failing - they're using different mock data than what I see with `MOCK_API=true`. Any thoughts on how to get `npm run start:integration` to use `MOCK_API=true`?

**Comment by bcaton85** [comment]
`MOCK_API: true` is already set in the CI [here](https://github.com/HammerMeetNail/quay-ui/blob/cypress_org_page/.github/workflows/ci.yml#L23). What is the difference in data that you are seeing?

File: cypress/e2e/org-list.cy.ts [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    cy.contains('Delete').click();
+    cy.contains('Permanently delete organizations?');
+    cy.contains(
+      'This action deletes all organizations and cannot be recovered.',
+    );
+    cy.contains('Confirm deletion by typing "confirm" below:');
+    cy.get('input[id="delete-confirmation-input"]').type('confirm');
+    cy.get('[id="bulk-delete-modal"]').within(() =>
+      cy.get('button:contains("Delete")').click(),
+    );
```
Comment (bcaton85) [inline cypress/e2e/org-list.cy.ts:92]: Do we want to assert the org no longer exists in the table?

File: cypress/e2e/org-list.cy.ts [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+    // Create Org
+    cy.get('#create-organization-button').click();
+    cy.get('#create-org-name-input').type('cypress');
+    cy.get('#create-org-email-input').type('cypress@redhat.com');
+    cy.get('#create-org-confirm').click();
+
+    // We fire too many requests with API v1, need to wait for server to respond
+    cy.wait(9000);
+    cy.visit('/organizations');
```
Comment (bcaton85) [inline cypress/e2e/org-list.cy.ts:40]: We should be able to remove the `cy.visit` here.

**Review by bcaton85 — APPROVED** [review]
LGTM - left two minor comments that i don't think are big issues.

**Pull request merged**